zad4/app/services/football_player_manager.rb

```diff
@@ -8,14 +8,16 @@ def initialize(player)
   end
 
   def stronger_players
-    # TODO: Implement me pls
+    Player.where('power > ?', @player.power)
```

Comment:
Uciekł Ci attribute_reader, został niewykrozystany, ale to nie jest błąd kodu ;)